apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/converter/TimeDurationValueConverter.java

```diff
@@ -50,6 +55,9 @@ public static TimeDurationValueConverter withDefaultFineDuration(String defaultD
     @Override
     public TimeDuration convert(String s) throws IllegalArgumentException {
         if (!s.endsWith("us") && !s.endsWith("ms") && !s.endsWith("s") && !s.endsWith("m")) {
+            if (defaultDurationSuffix == null) {
```

Comment:
Optional suggestion: Instead of allowing null values, you could also provide an empty string and let the validation within `TimeDuration` take care of the missing duration suffix.

Reply:
That is a good idea. I will change it.